cookiecutter.json

```diff
@@ -1,23 +1,20 @@
 {
     "project_name": "ProjectName",
-    "repo_name": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
-    "first_module_name": "{{ cookiecutter.repo_name.lower().replace(' ', '_') }}",
+    "repo_name": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_').replace(' ', '_') }}",
+    "github_username": "Your account name or organization name on GitHub where repo will be hosted",
+    "github_url": "{{ cookiecutter.github_username }}/{{ cookiecutter.repo_name }}",
+    "first_module_name": "{{ cookiecutter.repo_name.lower().replace(' ', '_').replace('-', '_').replace(' ', '_') }}",
     "author_name": "Your name (or your organization/company/team)",
     "author_email": "Your email (or your organization/company/team)",
-    "description": "A short description of the project (less than one line).",
-    "open_source_license": [
-
-        "MIT",
-        "BSD-3-Clause",
-        "LGPLv3",
-        "Not Open Source"
-    ],
+    "description": "A short description of the project.",
     "dependency_source": [
         "Prefer conda-forge over the default anaconda channel with pip fallback",
         "Prefer default anaconda channel with pip fallback",
         "Dependencies from pip only (no conda)"
     ],
```

Comment:
Do we still want to include these 3 options? They *are* all feasible, since MDAnalysis is on PyPI. That being said I'm still working on getting the cookie repo CI to pass on pip only on Mac https://github.com/lilyminium/test_mda_pip_rtd/

Reply:
yup all three, also the pip only MDA build runs fine on MacOS (tested several times), what issue are you getting?

Reply:
It's just pulling in 2.7 for some reason, I think I need a non-login bash shell

Reply:
link to runner? I can have a look in the morning

You should use one of the python setup actions if possible.

Reply:
Ah I fixed it by just not using a login bash shell :)